mydumper/mydumper_test.go

```diff
@@ -59,6 +60,25 @@ func (m *testMydumperSuite) TestArgs(c *C) {
 		"--password 123")
 	m.cfg.MydumperConfig.ExtraArgs = "--regex '^(?!(mysql|information_schema|performance_schema))'"
 	mydumper := NewMydumper(m.cfg)
-	args := mydumper.constructArgs()
+	args, err := mydumper.constructArgs()
+	c.Assert(err, IsNil)
 	c.Assert(args, DeepEquals, expected)
 }
+
+func (m *testMydumperSuite) TestEmptyExtraArgs(c *C) {
+	expected := strings.Fields("--host 127.0.0.1 --port 3306 --user root " +
+		"--outputdir ./dumped_data --threads 4 --chunk-filesize 64 --skip-tz-utc " +
+		"--tables-list mockDatabase.mockTable1,mockDatabase.mockTable2 " +
+		"--password 123")
+	m.cfg.MydumperConfig.ExtraArgs = ""
```

Comment:
also need add unit test for case:
1. ExtraArgs contain arg like "--skip-tz-utc", this case need generate table list
2. ExtraArgs contain arg like "-T" or || "--table-list" and so on, this case don't need generate table list